shell/common/shell_unittests.cc

```diff
@@ -148,6 +149,83 @@ class TestAssetResolver : public AssetResolver {
   AssetResolver::AssetResolverType type_;
 };
 
+class PlatformMessageResponseTest : public flutter::PlatformMessageResponse {
```

Comment:
I think both of these classes could be mocked with gmock which would be less code to maintain.